### PR TITLE
Molpro: Updated dtype Molpro in to_string 

### PR DESCRIPTION
**Pull request by sjrl**
Updated dtype Molpro in to_string to support symmetry and orientation molrec keys.

### CODE REVIEW

File: qcelemental/molparse/to_string.py [hunk 1]
```diff
@@ -152,13 +152,29 @@ def to_string(molrec: Dict,
...
+
+        # Don't orient the molecule if asked to fix_com or fix_orientation
+        if molrec['fix_orientation'] or molrec['fix_com']:
+            smol.append('{orient,noorient}')
+
+        # Have no symmetry if asked to fix_symmetry
+        if 'fix_symmetry' in molrec.keys() and molrec['fix_symmetry'] == 'c1':
+            smol.append('{symmetry,nosym}')
+        else:
+            smol.append('{symmetry,auto}')
```
Comment (loriab) [inline qcelemental/molparse/to_string.py]: aha, this is what I was remembering. fine -- not involved in the molecule.
```
mass,init,O=15.994914619560,H2=2.014101779,H3=3.01604927
{frequencies,analytic
thermo,sym=cs
print,thermo}
```

File: qcelemental/molparse/to_string.py [hunk 2]
```diff
@@ -152,13 +152,29 @@ def to_string(molrec: Dict,
...
+        smol = []
+
+        # Don't orient the molecule if asked to fix_com or fix_orientation
+        if molrec['fix_orientation'] or molrec['fix_com']:
+            smol.append('{orient,noorient}')
+
+        # Have no symmetry if asked to fix_symmetry
+        if 'fix_symmetry' in molrec.keys() and molrec['fix_symmetry'] == 'c1':
+            smol.append('{symmetry,nosym}')
+        else:
```
Comment (loriab) [inline qcelemental/molparse/to_string.py]: I think this is:
```
if 'fix_symmetry' not in molrec:
    smol.append('{symmetry,nosym}')
```

so the fix_symmetry, non-c1 case gets nothing. does this sound sensible to you?

Comment (sjrl) [reply qcelemental/molparse/to_string.py]: Yes that sounds reasonable to me.

File: qcelemental/molparse/to_string.py [hunk 3]
```diff
@@ -152,13 +152,27 @@ def to_string(molrec: Dict,
...
-        second_line = """geometry={"""
+        smol = []
+
+        # Don't orient the molecule if asked to fix_com or fix_orientation
+        if molrec['fix_orientation'] or molrec['fix_com']:
+            smol.append('{orient,noorient}')
+
+        # Have no symmetry if asked to fix_symmetry
+        if 'fix_symmetry' not in molrec.keys():
+            smol.append('{symmetry,nosym}')
```
Comment (loriab) [inline qcelemental/molparse/to_string.py]: sorry, I must have commented vaguely. _Two_ cases, I think: (a) 'fix_symmetry' present and == 'c1' --> `{symmetry,nosym}`, (b) 'fix_symmetry' absent --> `{symmetry,auto}`.

Comment (sjrl) [reply qcelemental/molparse/to_string.py]: Oh sorry, I misunderstood the previous comment. I've updated the code to handle the two cases.

Comment (sjrl) [reply qcelemental/molparse/to_string.py]: Oh and to clarify did you want the case for 'fix_symmetry' present and !='c1' to be given nothing right now?

Comment (loriab) [reply qcelemental/molparse/to_string.py]: Yeah, nothing to do for that case I think. There was sym=cs bellow but that’s for thermo, not for molecule.

**Pull request merged**